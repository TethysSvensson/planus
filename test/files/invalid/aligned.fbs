struct Aligned (force_align: 16) {
  v: uint8;
}

struct Aligned2 {
  v: uint8 (force_align: 16);
}
