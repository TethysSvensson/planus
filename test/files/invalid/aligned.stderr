error: Metadata attribute is not currently supported on structs
  ┌─ test/files/invalid/aligned.fbs:1:17
  │
1 │ struct Aligned (force_align: 16) {
  │                 ^^^^^^^^^^^^^^^

error: Metadata attribute does not make sense on struct fields
  ┌─ test/files/invalid/aligned.fbs:6:13
  │
6 │   v: uint8 (force_align: 16);
  │             ^^^^^^^^^^^^^^^

Error: Bailing because of errors
